### PR TITLE
add example to website

### PR DESCRIPTION
**Pull request by olaoluwasalami**
## Related issue number
Add new example

## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.
Screenshot 2023-12-04 at 07 58 00

### CODE REVIEW

**Comment by sonichi** [comment]
The gallery is a better place to add this example. Let's solve #850 first.

**Comment by olaoluwasalami** [comment]
> The gallery is a better place to add this example. Let's solve #850 first.

PR updated. added example to Gallery

File: website/docs/Examples.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Links to notebook examples:
    - Multimodal Agent Chat with DALLE and GPT-4V   - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_dalle_and_gpt4v.ipynb)
    - Multimodal Agent Chat with Llava  - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_lmm_llava.ipynb)
    - Multimodal Agent Chat with GPT-4V - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_lmm_gpt-4v.ipynb)
+   - Chainlit chat interface with multi-agent conversation between agents to complete a tasks - [View Notebook](https://github.com/antoineross/Autogen-UI/tree/main)
```
Comment (qingyun-wu) [inline website/docs/Examples.md:59]:
```suggestion
```

**Review by qingyun-wu — APPROVED** [review]
removing it from example

**Pull request merged**